### PR TITLE
Basic LocalDateTime support

### PR DESCRIPTION
**Pull request by gregh3269**
Add support for LocalDateTime for the LocalDateTime.now() variant. DateTimeFormatter.ISO_LOCAL_DATE_TIME.

See [WW-5175](https://issues.apache.org/jira/browse/WW-5175)

### CODE REVIEW

**Comment by lukaszlenart** [comment]
What's the status here? Do we still have a problem with `struts.date.format`?

**Comment by gregh3269** [comment]
Don't think so.  I did another patch (#3) to remove the !"".equals(globalFormat) check, and the globalDateFormat lookup on the LocalDateTime as it will break it.

**Comment by lukaszlenart** [comment]
Ok, so only tests need to be fixed

**Comment by gregh3269** [comment]
ok, I will have a look.

File: core/src/main/java/com/opensymphony/xwork2/conversion/impl/DateConverter.java [hunk 1]
```diff
@@ -156,20 +192,37 @@ private DateFormat[] getDateFormats(ActionContext context, Locale locale) {
...
 
+    /**
+     * Retrieves the list of date time formats to be used when converting dates
+     * 
+     * @param context the current ActionContext
+     * @param locale  the current locale of the action
+     * 
+     * @return a list of DateTimeFormatter to be used for date conversion
+     */
+    private DateTimeFormatter[] getDateTimeFormats(ActionContext context, Locale locale) {
```
Comment (lukaszlenart) [inline core/src/main/java/com/opensymphony/xwork2/conversion/impl/DateConverter.java:217]: Can this be `protected` to allow override?

**Review by lukaszlenart — COMMENTED** [review]
Looks good, just one suggestion

**Comment by lukaszlenart** [comment]
LGTM 👍

**Comment by gregh3269** [comment]
...maybe the formatting should be like java.sql.Timestamp, in line, as it only parses with a yyyy-mm-ddThh:mm:ss.  Early days here as its a big topic.

**Pull request merged**